### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by inverse**
- Fix some typos 
- Improve config access using `get()`
- Add `--verbose` flag to turn on debug level logging
- Simplify logger
- Improve shell scripts by adding interpreter and `+x`

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
     long_description = fh.read()
 
 setuptools.setup(
-    name='tell_me_your_secrets',
+    name='tell_me_your_secrets_tmp',
```
Comment (valayDave) [inline setup.py:6]: Module Name Change Required.

Comment (inverse) [reply setup.py:6]: Oh sorry - this was me debugging something when using the package in a project :)

Comment (inverse) [inline setup.py:6]:
```suggestion
    name='tell_me_your_secrets',
```

**Comment by inverse** [comment]
@valayDave updated :+1:

**Pull request merged**